backend-rust/src/graphql_api/block_metrics.rs

```diff
@@ -236,8 +232,6 @@ LEFT JOIN LATERAL (
                 .total_amount_released
                 .unwrap_or(0)
                 .try_into()?,
-            last_total_micro_ccd_unlocked: None, // TODO implement unlocking schedule
```

Comment:
Does the frontend still query for this field? because then we need to fix that as well otherwise it will break

Reply:
The dependency was removed in this PR `https://github.com/Concordium/concordium-scan/pull/531`. This means that we need to roll out the changes to the frontend first and then update api services.

Reply:
I have rolled out the frontend